### DISCUSSION TITLE
video:motion_type + avoid "FMV"

### DISCUSSION THREAD

**Discussion by bradh** [stac-extensions/video#7] [General]
There is a discussion of Full Motion Video as a term (and lack of good definition) in the MISB Motion Imagery Handbook. Its a somewhat emotive issue, and is generally taken as having a sufficient frame rate to look like moving pictures.

Its definitely not what `video:motion_type` refers to as full. If we want to handle the case where the field of view is stationary, I think that might be better captured as `video:constant_scene` or something like that. I'm also a bit cautious that determining that might be hard in that you'd need to compare the metadata for a whole file to check. 

Possibly this doesn't matter - I really only care that there is some intersection of the location / time window that I'm interested in with the video. If things move before or after, I'm still interested.

**Comment by darrenwiens** [comment]
Ah, that's good to know. I was thinking of FMV meaning that the sensor moved.

The aim of `video:motion_type` is to avoid forcing unnecessary geojson file(s) that all contain the same value. Happy to switch it to `video:contant_scene` to avoid confusion.

**Reply by darrenwiens** [reply]
Addressed in: https://github.com/sparkgeo/stac-video/pull/12
